.env.example

```diff
@@ -0,0 +1,80 @@
+AMAZON_EN_URL=''
```

Comment:
We already have [.env.development](https://github.com/annict/annict/blob/74884acbd987f11a08c71aff97dc58dea37b9fdc/.env.development). Why does this need?

Reply:
This is for hosting Annict on somebody's own hardware.

Reply:
hmm, I'm not very interest that someone runs Annict on their server. If necessary, they can create it. I don't use this file, so I will probably not manage this file.